doc/design/paths-endpoint-webdav.md

```diff
@@ -0,0 +1,21 @@
+# Discussion on metadata needed for dandidav
+
+Submitted as a PR to avoid spagetti thread discussion on https://github.com/dandi/dandi-archive/issues/1837#issuecomment-1921864949
+
+
+@jwodder listed needed properties and metadata fields:
+
+* Asset properties used by dandidav:
+    * `asset_id` (PRESENT)
+    * `blob_id`
```

Comment:
what for do we need blob_id if we have contentUrl?

Reply:
The `contentUrl` returned in `.../assets/paths/` responses is a plain S3 URL, and serving it to the user in a web browser would result in the file being downloaded and named with the blob ID.  In order for the downloaded file to be named with the asset's filename instead, we need a signed S3 URL, which is obtained via the API download URL in the `contentUrl` field of the metadata.

Reply:
ok, so we the actually need `result_type` (Folder, AssetBlob, AssetZarr) as I suggested in https://github.com/dandi/dandi-archive/issues/1837#issuecomment-1921643799 .